env/includes/settings/apps/tk-multi-publish2.yml

```diff
@@ -79,3 +84,16 @@ settings.tk-multi-publish2.ingest:
   path_info: '@settings.tk-multi-publish2.path_info'
   publish_plugins: '@settings.tk-multi-publish2.ingest.publish_plugins'
   location: '@apps.tk-multi-publish2.location'
+
+  # Library
+settings.tk-multi-publish2.library:
+  display_name: 'Ingest'
```

Comment:
"Library Ingest" to distinguish it from regular "Ingest"?

Reply:
Shotgun Ingest wrapper expects a command that's registered with name "Ingest", we can't change this name, the wrapper will not run.

```
    try:
        return engine.execute_command("Ingest...", [])
    except Exception as e:
        logger.error("Could not start app: %s\nThis app is supposed to be launched from ingest role." % str(e))
        if os.environ.get("DD_DEBUG"):
            import pdb; pdb.post_mortem()
        raise
```